.circleci/config.yml

```diff
@@ -373,10 +373,6 @@ jobs:
 workflows:
   version: 2
   commit:
-    unless: &packaging_cond
```

Comment:
I'm not against, but I think I original added those because the team was annoyed by CI failures when they added something that broke packaging

Reply:
Dude, this is from the REMOVE commit.

Reply:
Oh, carry on , then